modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/filename/PdsFolderSettings.java

```diff
@@ -28,7 +28,7 @@
 /**
  * Class holds information required for folder generation for ignite persistent store
  */
-public class PdsFolderSettings {
+public class PdsFolderSettings<FLH extends FileLockHolder> {
```

Comment:
According to the [Type Parameter Naming Conventions](https://docs.oracle.com/javase/tutorial/java/generics/types.html) the `FLH` abbreviation is good to be a single upper-case letter only.

Reply:
Renamed to `L`